content/posts/newsletter-013/index.md

```diff
@@ -301,6 +301,29 @@ Jazarro on [the Amethyst discord][amethyst-discord].
 [kv-ii]: https://en.wikipedia.org/wiki/King%27s_Valley_II
 [amethyst-discord]: https://discord.com/invite/amethyst
 
+### [Akigi][akigi]
+
+[Akigi][akigi] is a magical multiplayer online world where humans aren't the
```

Comment:
This actually isn't the tag line / theme of the game anymore - although to be fair it's been changing every few months ...

Lately it's just been `The multiplayer online world of Akigi.`

Reply:
Updated!